Sources/Player/UserInterface/AVPlayerViewControllerSpeedCoordinator~tvos.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import AVKit
+import Combine
+import UIKit
+
+@available(iOS, unavailable)
+class AVPlayerViewControllerSpeedCoordinator {
+    let player: Player
+    let controller: AVPlayerViewController
+    private var cancellables = Set<AnyCancellable>()
+
+    init(player: Player, controller: AVPlayerViewController) {
+        self.player = player
+        self.controller = controller
+        configurePlaybackSpeedPublisher(player: player, controller: controller)
+    }
+
+    private func configurePlaybackSpeedPublisher(player: Player, controller: AVPlayerViewController) {
+        Publishers.CombineLatest(
+            player.playbackSpeedUpdatePublisher(),
+            player.$_playbackSpeed
```

Comment:
`$_playbackSpeed` is derived from `playbackSpeedUpdatePublisher()` so this seems redundant, especially if the goal is to avoid duplication removal. I think we could think of another way to write this publisher.

Reply:
What do you think about the following suggestion?

```swift
player.$_playbackSpeed
            .receiveOnMainThread()
            .sink { speed in
                guard let range = speed.range else { return }
                if range == 1...1 {
                    controller.transportBarCustomMenuItems = []
                }
                else {
                    Self.configureSpeeds(for: player, controller: controller, range: range, speed: speed.effectiveValue)
                }
            }
            .store(in: &cancellables)
```

Reply:
Would be possible, though I would suggest using the underlying publisher rather than a published property publisher where possible. 

I probably miss the big picture a bit here, though. Maybe it's best we discuss this tomorrow morning so that I can have a deeper understanding of the problems you faced during the implementation.